actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -235,6 +235,20 @@ def method_missing(called, *args, **options, &block)
       #   # A void element:
       #   tag.br  # => <br>
       #
+      # === Building HTML attributes
+      #
+      # Transform a Hash into HTML Attributes, ready to be interpolated into
```

Comment:
Should the verb tense be:

```suggestion
      # Transforms a Hash into HTML attributes, ready to be interpolated into
```

to match the other lines?